client/elife-theme/src/elements/ui/Menu.js

```diff
@@ -0,0 +1,51 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+export default {
+  Arrow: css`
+    color: ${th('colorTextSecondary')};
+    transition: none;
+    font-size: ${th('gridUnit')};
```

Comment:
space?

Reply:
Having 2 variables in our theme which evaluate to the same thing (`gridUnit` and `space.1`) seems less than ideal to me. Also for `calc(...something based on gridUnit...)` vs `space.4`/`space.3`/whatever.

Seems like we need a team-wide convention. Perhaps `gridUnit` is fine for now @dryhten?